### ISSUE TITLE
ResNet50 architecture

### ISSUE THREAD

**Issue by nhewadehigah** [facebookresearch/dinov2#438]
Hi is DINOv2 contains ResNet50 model? I found it in DINO but could not found in DINOv2. Thank you.

**Comment by crypdick** [comment]
No. DinoV2 requires patch tokens and register tokens, which you can't get out of a pure convnet.
